c++/src/Timezone.cc

```diff
@@ -565,7 +565,7 @@ namespace orc {
 
   class TimezoneImpl : public Timezone {
    public:
-    TimezoneImpl(const std::string& name, const std::vector<unsigned char> bytes);
+    TimezoneImpl(const std::string& name, const std::vector<unsigned char>& bytes);
```

Comment:
nit: the function parameter names are inconsistent between declaration and definition. It would be nice to make them consistent.

Reply:
+1 for the above comment.

Reply:
yeah, resolved.